src/routers/index.js

```diff
@@ -1,9 +1,27 @@
 import { Router } from 'express';
+import { AddArticles, UpdateArticle, DeleteArticle } from '../controllers/article';
 
 const router = Router();
 
 router.get('/', (req, res) => res.status(200).json({
   message: 'Welcome to the Authors Haven API',
 }));
 
+/**
+ * Resource handling articles
+ * @name router:/articles/:id?
+ * @function
+ * @memberof module:Express.Router
+ * @inner
+ * @param {function} AddArticles - Express path
+ * @param {function} DeleteArticle - Express path
+ * @param {function} UpdateArticle - Express path
+ * @returns Response Object
+ */
+router
+  .route('/articles/:id?')
+  .post(AddArticles)
+  .delete(DeleteArticle)
```

Comment:
@chikeozulumbaI think to delete article is out of the scope of this ticket

Reply:
@chikeozulumba I didn't implement it within the PR, they all have their stories assigned to them, same response to all your comments